### ISSUE TITLE
installation issues in spacyr

### ISSUE THREAD

**Issue by alexstark001** [quanteda/spacyr#236]
I was not able to install the spacey package in r. I am getting the following error when initializing.

spacy_initialize()
Found 'spacy_condaenv'. spacyr will use this environment
Error in py_run_file_impl(file, local, convert) :
ModuleNotFoundError: No module named 'spacy'
![Capture](https://user-images.githubusercontent.com/122796881/212684406-e25fbb38-3152-4dae-8deb-3482edf6461a.PNG)

**Comment by kbenoit** [comment]
Can you supply any information on your system? 

@Mayazure any ideas?

**Comment by JBGruber** [comment]
I was stuck on the solving environment part you show in your screenshot this morning. What solved it was to change `python_version` in  `spacy_install` to the actual version displayed by `reticulate::py_config()[["version"]]`:

```
spacyr::spacy_install(python_version = reticulate::py_config()[["version"]])
```

Different system, different background (this was on Debian bullseye; you seem to be on Windows). But worth a try, I think.

**Comment by kbenoit** [comment]
Thanks @JBGruber that's exactly the solution, or simply specifying your current version of Python as a string. This is an issue we have to address, since 3.6 is hardwired into `spacy_install()` as a default value.
e.g.
https://github.com/quanteda/spacyr/blob/f9300724e117b7e8eb593aa6a566ef7cc742a943/R/spacy_install.R#L63

@amatsuo we discussed refreshing this a few months ago. I'm too time-poor at the moment to get lost in this (although I would love to do so). Maybe @Mayazure could address it?

**Comment by amatsuo** [comment]
@leodumont 

What happens when you try to run spacy from your terminal. 

In terminal, could you try the following and show the output in each step?

```
## activate the environment
conda activate spacy_condaenv
## check the python version
python --version
## also the path 
which python 
## start python
python

## in python
import spacy 

nlp = spacy.load("en_core_web_sm")

```

**Comment by chainsawriot** [comment]
I had the same problem. My solution is too build the conda environment myself (Ubuntu 22.04).

```sh
## remove the (obsolete) Python 3.6 environment
#conda env remove --name spacy_condaenv
## install the "recommended" version of Python
Rscript -e 'reticulate::conda_create("spacy_condaenv")'

## install spacy manually
conda install -n spacy_condaenv spacy -y  
conda activate spacy_condaenv
python -m spacy download en_core_web_sm 
conda activate base
```

```R
require(spacyr)
spacy_initialize()
spacy_parse("I solved this by building my own conda environment.")
```

**Comment by marcozanotti** [comment]
Very same problem here on 

R version 4.2.2 Patched (2022-11-10 r83330)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.2 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

The solution proposed by @JBGruber did not work, unfortunately. Instead, the solution proposed by @chainsawriot worked. I managed it completely through R using reticulate instead of using conda from cli. Here is my workaround

```
reticulate::conda_remove("spacy_condaenv")  
reticulate::conda_list()  
reticulate::conda_create("spacy_condaenv")  
reticulate::conda_install("spacy_condaenv", packages = "spacy")  
spacyr::spacy_download_langmodel(model = "en")    
```

Then proceeding as usual with the language initialization.

**Comment by MonicaDiLeo** [comment]
I had the same issue. The solution posted by @marcozanotti worked for me. Thanks!

**Comment by JBGruber** [comment]
I rewrote `spacy_install()` from scratch. It's not in the repo yet, but there is a pull request that you can test:

```r
# install packaged from fork
remotes::install_github("JBGruber/spacyr")

# install spaCy and initialize
library(spacyr)
spacy_install()
spacy_initialize()

# test
txt <- c(d1 = "spaCy is great at fast natural language processing.",
         d2 = "Mr. Smith spent two years in North Carolina.")

parsedtxt <- spacy_parse(txt)
parsedtxt
```

I also included a file for debugging and manual/custom installation that might be useful: https://github.com/JBGruber/spacyr/blob/master/installation.Rmd.

**Comment by MarianoRico** [comment]
Please, @JBGruber, could you update the CRAN version? I guess many people is waiting for this major update.

**Comment by kbenoit** [comment]
Should be by early next week. Adding tests and even building the documentation for **spacyr** is not trivial, but we're getting there soon!

**Comment by kbenoit** [comment]
Solved in v1.3.
